cypress/locators/show-edit-pod/locators.js

```diff
@@ -1,9 +1,16 @@
 // component preview locators
-export const SHOW_EDIT_POD_CONTENT_BOX = '[data-element="content"]';
-export const SHOW_EDIT_POD_CONTENT_ELEMENTS = '[data-component="content"]';
-
-export const SHOW_EDIT_POD_TITLE = "h4[data-element=title]";
-export const SHOW_EDIT_SAVE_BUTTON = 'button[data-element="submit-button"]';
-export const SHOW_EDIT_CANCEL_BUTTON = 'button[data-element="cancel-button"]';
-export const SHOW_EDIT_DELETE_BUTTON = "[data-element=delete-button]";
-export const SHOW_EDIT_POD_EDIT = '[data-element="edit"]';
+export const SHOW_EDIT_POD = '[data-component="pod"]';
+export const CONTENT_SHOW_EDIT_POD = '[data-component="content"]';
+export const CONTENT_FORM_SHOW_EDIT_POD = '[data-element="form-content"]';
+export const HIDE_DELETE_BUTTON_SHOW_EDIT_POD = "button[data-element=delete]";
+export const EDIT_FIELDS_SHOW_EDIT_POD = '[data-element="input"]';
+export const FORM_FOOTER_SHOW_EDIT_POD = '[data-element="form-footer"]';
```

Comment:
there is such locator in `common` locators

Reply:
we don't have "form-footer" in common, I found content and input.